07-graphs/__test__/graph.test.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const Graph = require('../lib/graph_traverse');
+
```

Comment:
The most important test for these graph/tree traversals is absent. That is, testing that the order of the nodes (in this case breadth first) are traversed in the order that we expect. Since that is the method's purpose/core functionality, we want to make sure we test it.